doc/api.rst

```diff
@@ -114,21 +113,40 @@ The Job class
     :show-inheritance:
 
 
+.. currentmodule:: signac
+
 The Collection
 ==============
 
-.. currentmodule:: signac.contrib.collection
-
 .. autoclass:: Collection
    :members:
 
+
+The H5Store
+===========
+
+This class implements the interface to the job's :attr:`~signac.contrib.job.Job.data` attribute, but can also be used stand-alone:
+
+.. autoclass:: H5Store
+    :members:
+
```

Comment:
I'm not sure if this affects anything but it looks like there is one empty line here, compared to two empty lines before "The H5Store" and after "The H5StoreManager"

Reply:
It doesn't look like it, but either way it wouldn't matter.